docs/reference/commandline/kill.md

```diff
@@ -27,8 +27,11 @@ Options:
 
 ## Description
 
-The main process inside the container will be sent `SIGKILL`, or any
-signal specified with option `--signal`.
+The `docker kill` subcommand kills one or more containers. The main process
```

Comment:
Maybe not for this PR but do we talk anywhere about the nuances between "stop", "terminate", "kill" etc, as concepts?